.github/workflows/build.yml

```diff
@@ -68,6 +68,15 @@ jobs:
             tags: |-
               jrei/systemd-debian:12
               jrei/systemd-debian:bookworm
+          - name: systemd-debian-13
```

Comment:
`jrei/systemd-debian:latest` has to be moved above this line to have latest = 12